packages/react-app/src/InkCanvas.js

```diff
@@ -256,7 +256,45 @@ const onFinishFailed = errorInfo => {
   console.log('Failed:', errorInfo);
 };
 
+const fillBackground = (color) => {
```

Comment:
Nice. I wonder if there is a way to inject this so it's a true background (currently it's definitely a fill, but it kind of fills in everything)

Reply:
maybe that could be possible if we forked react-canvas-draw and changed every code from

```
ctx.clearRect(0, 0, ctx.canvas.width, ctx.canvas.height);
```

to fill the color we passed (there is no option to pass either)

```
ctx.fillStyle = FILL_COLOR;
ctx.fillRect(0, 0, ctx.canvas.width, ctx.canvas.height);
```

I think Osama's approach is better, very nice workaround.

Reply:
I guess I meant more to show up behind any drawing that has already been done. But there is a lot of complexity there I think

Reply:
Actually there can be an easy way to make this always stay on the background. Maybe I can check it this evening if that works. basically if we manage to always draw background before everything else that would solve our problem